db/seeds.rb

```diff
@@ -1,6 +1,11 @@
 require 'faker'
 
-10.times { User.create!(name: Faker::Name.name) }
+10.times do 
+  User.create!(name: Faker::Name.name,
+               email: Faker::Internet.email,
+               password: Faker::Internet.password) 
+end
+
 loop do 
```

Comment:
Either use `15.times` instead of loop or use `create!` instead of `create` to prevent involuntary infinite loop.